tests/regionizers/test_administrative_boundary_regionizer.py

```diff
@@ -57,35 +59,40 @@ def test_no_crs_gdf_value_error(gdf_no_crs) -> None:  # type: ignore
         abr.transform(gdf=gdf_no_crs)
 
 
-@pytest.mark.parametrize(  # type: ignore
-    "toposimplify",
-    [
-        (True),
-        (0.0001),
-        (0.001),
-        (0.01),
-        (False),
-        (0),
-    ],
-)
-def test_single_points(toposimplify: Union[bool, float]) -> None:
-    """Test checks if single points work."""
-    country_points_gdf = gpd.GeoDataFrame(
+@pytest.fixture  # type: ignore
+def mock_for_madagascar(mocker: MockerFixture) -> None:
```

Comment:
Since we have changed the names and geometry, it probably shouldn't be named after Madagascar and we can arbitrarily modify values.

Reply:
Consider it done ;)